packages/domain-picker/src/domain-picker/suggestion-item.tsx

```diff
@@ -59,6 +59,10 @@ const DomainPickerSuggestionItem: FunctionComponent< Props > = ( {
 
 	const labelId = uuid();
 
+	const learnMoreUrl = () => {
+		return `https://wordpress.com/${ i18nLocale }/support/https-ssl`;
```

Comment:
WIP: this works like a charm for logged-in user (tested in es and de)

Need a solution for logged out user as this doesn't register browser settings language.

Reply:
We only have support docs for mag 16 locales, this really needs to use the `localizeUrl` function so that the english fallback is passed back when the locale isn't covered. 

Using i18nLocale also has problems, its the .org locale which does not map to .com locales, or .com url slugs (sometimes different again.)

Logged out users should already be handled by your code here, they will arrive on the right url that sets the state stored locale.

Browser settings language can be ignored in most a8c contexts, there are very few situations they make sense for our apps.

Reply:
Parking this for now.
Context: p1603433729140700-slack-CRWCHQGUB
And waiting for this language package: https://github.com/Automattic/wp-calypso/pull/46709